flac.go

```diff
@@ -51,18 +51,24 @@ func ReadFLACTags(r io.ReadSeeker) (Metadata, error) {
 	return m, nil
 }
 
-type metadataFLAC struct {
+type MetadataFLAC struct {
```

Comment:
This should remain unexported (we don't want to widen the surface area of the API)